views/dashboard-resources.hbs

```diff
@@ -0,0 +1,203 @@
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+
+    <!-- GOOGLE FONTS -->
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;1,700&family=Source+Code+Pro:wght@700&display=swap" rel="stylesheet">
+    <!-- STYLESHEETS -->
+    <link rel='stylesheet' type='text/css' href='/css/style.css' />
+    <link rel='stylesheet' type='text/css' href='/css/dashboard-resources.css' />
+    <!-- SCRIPTS -->
+    <script src='/js/boxes.js'></script>
+</head>
+<body>
+
+    <div class="wrapper">
+        <!-- NAVBAR -->
+        {{> navbar }}
+        <!-- RESOURCES TITLE-->
+        <div class="container text-center resource-title">
+            <h1 class="Resources">Resources</h1>
+        </div>
+        <!--Main Container-->
+        <div class="container-fluid">
+            <!-- Base Parts -->
+            <div class="row">
+                <div class="col-md-12 text-center">
+                    <h3 class="base-box">BASE PARTS</h3>
+                </div>
+            </div>
+            <!--row 1 -->
+            <div class="row">
+                <div class="col-md-6">
+                    <iframe src="https://www.youtube.com/embed/F_vLWkkOETI" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>             
+                </div>
+                <div class="col-md-6">
+                    <iframe src="https://www.youtube.com/embed/6WReFkfrUIk" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                </div>
+            </div>
+            <!--row 2 -->
+            <div class="row">
+                <div class="col-md-6 ">
+                    <iframe src="https://www.youtube.com/embed/qEMtCKfZOHw" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                </div>
+                <div class="col-md-6 ">
+                    <iframe src="https://www.youtube.com/embed/6IAkYpmA1DQ" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                </div>
+            </div>
+            <!--row 3 -->
+            <div class="row">
+                <div class="col-md-6 ">
+                    <iframe src="https://www.youtube.com/embed/xPlN_Tk3VLQ" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                </div>
+            </div>
+            <!--row 4 -->
+            <div class="row text-center">
+                <div class="col-md-6">
+                    <p>Arduino Foundations &nbsp

```

Comment:
Is having %nbsp necessary?

Reply:
my workaround to adding a space between text and link